projects/pastanaga-angular/src/lib/modal/modal.model.ts

```diff
@@ -19,6 +19,7 @@ export class ModalRef {
     id: number;
     isLast: boolean;
     config: ModalConfig;
+    data?: any;
```

Comment:
ok so that is good but you also need to update pastanaga documentation about passing data to a modal.
The documentation is in `projects/demo/src/app/demo/pages/modal-page/modal-page.component.html`.

Reply:
Other modals I've used in the past allowed for generic typing.  Is that something we could consider?

```
export class ModalRef<T = any> {
  ...
  data: T;
  ...
}
```

Reply:
I update per Daniel's suggestion... I also added the additional update to the atlas demo and included the service update in a separate commit.